adobecom/milo
# [MWPW-157973] Run Nala checks on all PRs by default

## Description
This PR updates the Git workflow to run Nala checks on all PRs by default.

## Related Issue
Resolves: [MWPW-157973](https://jira.corp.adobe.com/browse/MWPW-157973)

## Testing instructions
See that the Nala Tests are running on this PR without having the `run-nala` label added.

## PSI check URL
https://mwpw-157973-nala-by-default--milo--robert-bogos.hlx.page/drafts/rbogos/page-default?martech=off
